charts/doccano/templates/ingress.yaml

```diff
@@ -21,11 +21,12 @@ spec:
 {{ toYaml .Values.ingress.tls | indent 4 }}
 {{- end }}
   rules:
+{{- $path := .Values.ingress.path -}}
```

Comment:
As we do a `{{- range .Values.ingress.hostnames }}` it seems that we cannot access .Values below line 25.

If you are good with this solution we can live it there, or we can also put this line on top of the file, what do you think ?

Reply:
Yep that's right, you can only access values that are within the ranged object like in a for you have variable scoping, and you cannot access variables that are out of scope.
I prefer that you put it on top with the others.

Reply:
You should also increase the Chart version from 1.3.6 to -> 1.3.7 in the `chart.yaml` thanks.